src/image-display.cc

```diff
@@ -120,6 +120,23 @@ static ExifImageOp GetExifOp(Magick::Image &img) {  // NOLINT
     return {};
 }
 
+// An extended version of Magick::readImages that requests a
+// decoding/raster with a transparent background.
+static void readImagesWithTransparentBackground(
+    std::vector<Magick::Image> *sequence, const std::string &image_spec) {
+    MagickLib::ImageInfo *image_info = MagickLib::CloneImageInfo(0);
+    // Set the opacity quantum to maximum value for transparent background.
+    image_info->background_color.opacity = ((Magick::Quantum)(257U * (255)));
```

Comment:
this is an odd way of writing 65535, and seems to assume that you already know the size of quantum; but that is implementation dependent.

I think it would be be more portable if you use `ScaleCharToQuantum(255)`, because then the correct macro is used depending on the image magic installation.

Reply:
(solved)